### PR TITLE
Fixes #197

### LINKED ISSUE #197: Alignment issue in payment successful screen.
# Bug Report 🐛
The transform.translate is not a readable code. The 'continue shopping' is also not working. The alignment of the UI elements is not proper. 

## Expected Behavior
The UI should be properly aligned, and code should be readable. 

## Current Behavior
The payment successful screen's UI is not proper, as you can see in the screenshot. 

WhatsApp Image 2021-04-20 at 06 22 24 (2)

### PR DESCRIPTION
**Pull request by SapinderSingh**
### Related Issue
- Fixes #197 .

### Proposed Changes
- Fixed payment successful screen.

### Additional Information
- Any additional information about the change

### Checklist
- [X] Tested
- [X] No Conflicts
- [X] Change In Code
- [ ] Change In Documentation
- [X] Follows linting rules

### Screenshots

**Before this PR**    
WhatsApp Image 2021-04-20 at 06 22 24 (2)

**After this PR**
WhatsApp Image 2021-04-20 at 13 12 54

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -58,6 +58,7 @@ flutter:
   assets:
     - assets/
     - assets/items/
+    - images/
```
Comment (himanshusharma89) [inline pubspec.yaml:61]: don't add images here

File: lib/helpers/constants.dart [hunk 2]
```diff
@@ -93,6 +93,3 @@ List<ProductCard> productsList = <ProductCard>[
         height: 29),
   ),
 ];
-
-const String svgIv5cq2 =
```
Comment (himanshusharma89) [inline lib/helpers/constants.dart:97]: Use this SVG instead of an image, add this in constants

**Review by himanshusharma89 — CHANGES_REQUESTED** [review]

**Comment by himanshusharma89** [comment]
@SapinderSingh Delete the bulb image

**Comment by SapinderSingh** [comment]
> @SapinderSingh Delete the bulb image

Done

**Pull request merged**